turbopack/crates/turbopack-ecmascript/src/references/esm/export.rs

```diff
@@ -429,6 +430,7 @@ async fn emit_star_exports_issue(source_ident: Vc<AssetIdent>, message: RcStr) -
 #[turbo_tasks::value(shared)]
 #[derive(Hash, Debug)]
 pub struct EsmExports {
+    pub parsed: ResolvedVc<ParseResult>,
```

Comment:
Instead of storing that here, we should instead pass the `ParseResult` to `code_generation`